index.html

```diff
@@ -27,6 +41,7 @@
 	<div class="home-body">
 		
 		<h1 class="chloe-smart">Chloe Smart</h1>
+		<p class="software-developer">SoftwareDeveloper</p>
```

Comment:
this p needs additional sizing when viewing as a mobile view, as it is overlapping with H1 when the screen size is set to 400px